lib/upload_creator.rb

```diff
@@ -406,8 +406,8 @@ def clean_svg!
       if use_el.attr('href')
         use_el.remove_attribute('href') unless use_el.attr('href').starts_with?('#')
       end
-      if use_el.attr('xlink:href')
-        use_el.remove_attribute('xlink:href') unless use_el.attr('xlink:href').starts_with?('#')
+      if use_el.attr('href')
```

Comment:
This is unnecessary cause we have the exact same check above :) I actually think maybe this could stay as is in case someone is uploading an SVG with that?

Reply:
Good catch!! It's entirely the same